dataprofiler/data_readers/data_utils.py

```diff
@@ -843,3 +846,138 @@ def url_to_bytes(url_as_string: Url, options: Dict) -> BytesIO:
 
     stream.seek(0)
     return stream
+
+
+class S3Helper:
+    """
+    A utility class for working with Amazon S3.
+
+    This class provides methods to check if a path is an S3 URI
+        and to create an S3 client.
+
+    Attributes:
+        None
+
```

Comment:
can probably remove this entirely?

Reply:
As you requested, deleted extra details from docstring.